compiler-plugin/src/main/java/io/ballerina/stdlib/email/compiler/EmailServiceValidator.java

```diff
@@ -276,6 +279,24 @@ private void validateParameter(SeparatedNodeList<ParameterNode> parameterNodes,
         }
     }
 
+    private boolean validateOnMessageParams(FunctionDefinitionNode functionDefinitionNode) {
+        FunctionTypeSymbol functionTypeSymbol = ((MethodSymbol) ctx.semanticModel().symbol(functionDefinitionNode)
+                .get()).typeDescriptor();
+        List<ParameterSymbol> inputParams = functionTypeSymbol.params().get();
```

Comment:
Better to check `isPresent()` before calling `get()`.

Reply:
Added the check.